charts/backend/templates/polling-configmap.yaml

```diff
@@ -21,4 +33,5 @@ data:
   ACTIVATED_DEPOSITS_NOTIFY_SECONDS: {{ .Values.polling.conf.activatedDepositsNotifySeconds | quote }}
```

Comment:
REALITY_ETH_QUESTION_URL_TEMPLATE, VOTING_PAGE, ACTIVATED_DEPOSITS_NOTIFY_SECONDS were moved to base_settings, they must be in common configmap